hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -0,0 +1,115 @@
+# Pi4 Bootloader Configuration
+
+You can display the current configuration using 
+```
+vcgencmd bootloader_config
+```
+
+To change these bootloader configuration items, you need to extract the configuration segment, make changes, re-insert it, then reprogram the EEPROM with the new bootloader. 
+
+```
+# Extract the configuration file
+cp /lib/firmware/raspberrypi/bootloader/beta/pieeprom-2019-09-10.bin pieeprom.bin
+rpi-eeprom-config pieeprom.bin > bootconf.txt
+
+# Edit the configuration using a text editor e.g. nano bootconf.txt
+
+# If you have a UART cable then setting BOOT_UART=1 will help debug boot issues
+BOOT_UART=1
+
+# Save the new configuration and exit editor
+
+# Apply the configuration change to the EEPROM image file
+rpi-eeprom-config --out pieeprom-new.bin --config bootconf.txt pieeprom.bin
```

Comment:
OffTopic: Maybe it's just me, but I keep reading "pieeprom" as "Pie Eprom" instead of "Pi Eeprom" :laughing:

Reply:
me too..